### ISSUE TITLE
Unity WebGL2 not working (Built-in RP)

### ISSUE THREAD

**Issue by AdrianTyrannix** [rive-app/rive-unity#54]
Hi, I'm unable to get Rive working in webgl, if it does would, could you tell me the package version and unity version that works for webgl?

I have gotten everything working in the editor and modified the scripts to render rive textures into the Sprite Renderer/Raw Texture for UI, built the game but got a surprise error when building for webgl.

I am using 
https://github.com/rive-app/rive-unity.git?path=package#v0.1.125
since that's the latest version I was able to get running for my testing, any other "more latest" version, crashes my editor (Unity 2022.3.21f1)

In webgl, I get this error after making the build:

![Screenshot_5](https://github.com/rive-app/rive-unity/assets/166386525/9dbbc027-ee9d-4220-b0e1-b1b534a92e57)

While trying to build the example scenes provided (v0.1.69), I get this error:

Building Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js failed with output:
emcc: error: Library/PackageCache/app.rive.rive-unity@ba61818f2a/Runtime/Libraries/iOS/rive.a: Unknown format, not a static library!

UnityEditor.BuildPlayerWindow:BuildPlayerAndRun ()

Am I doing anything wrong or is webgl not working / broken right now (seeing as its listed as supported on the docs)?

Thanks

**Comment by damzobridge** [comment]
Hi Adrian,

We're still looking into the `randomWrites` bug, but I wanted to let you know that the following build error has been addressed in [v0.1.169](https://github.com/rive-app/rive-unity/releases/tag/v0.1.169) upwards:

```
Building Library\Bee\artifacts\WebGL\build\debug_WebGL_wasm\build.js failed with output:
emcc: error: Library/PackageCache/app.rive.rive-unity@ba61818f2a/Runtime/Libraries/iOS/rive.a: Unknown format, not a static library!
```

Will let you know once we've solved the URP black texture bug.

**Comment by damzobridge** [comment]
Hi @AdrianTyrannix , we're working on a larger update that will include high-level components that handle rendering across built-in RP, URP, and HDRP. 

In the meantime, the updated [RiveTexture script here](https://github.com/rive-app/rive-unity-examples/blob/main/getting-started/Assets/Rive/RiveTexture.cs) should address the `randomWrites` problem.
